src/components/avatar-audio-source.js

```diff
@@ -0,0 +1,115 @@
+const INFO_INIT_FAILED = "Failed to initialize avatar-audio-source.";
```

Comment:
For reference, here is `networked-audio-source` : https://github.com/MozillaReality/networked-aframe/blob/8d0dc7355c1c72ba0bb49a20e5920da0a0a92719/src/components/networked-audio-source.js#L1

I also put a diff up in this gist, but they're pretty different so it's probably easier to just read this new version. https://gist.github.com/johnshaughnessy/33517261b58f36d393ca740829f40e1e/revisions